config/config.go

```diff
@@ -62,14 +62,18 @@ func getConfigFiles() []string {
 	env := Environment()
 	dc := os.Getenv(EnvironmentPrefix() + _datacenter)
 
-	var files []string
+	baseFiles := []string{_baseFile, env, _secretsFile}
 	if dc != "" && env != "" {
-		files = append(files, fmt.Sprintf("./%s/%s-%s.yaml", _config, env, dc))
+		baseFiles = append(baseFiles, fmt.Sprintf("%s-%s", env, dc))
+	}
+
+	var files []string
+	dirs := []string{".", _configRoot}
```

Comment:
Question: Why "." you will be adding ./base.yaml then ./config/base.yaml etc.. won't the lookup in _configRoot be sufficient?